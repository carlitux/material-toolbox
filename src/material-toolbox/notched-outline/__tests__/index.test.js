// @flow
import React from 'react';
import { shallow } from 'enzyme';
import NotchedOutline from '../index';

describe('component::NotchedOutline', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<NotchedOutline />);
    expect(component).toMatchSnapshot();
  });

  it('Should match snapshoot cssOnly', () => {
    const component = shallow(<NotchedOutline cssOnly />);
    expect(component).toMatchSnapshot();
  });
});
