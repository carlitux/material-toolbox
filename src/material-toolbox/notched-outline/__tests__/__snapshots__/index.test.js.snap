// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component::NotchedOutline Should match snapshoot 1`] = `
<React.Fragment>
  <div
    className="mdc-notched-outline"
  >
    <svg>
      <path
        className="mdc-notched-outline__path"
      />
    </svg>
  </div>
  <div
    className="mdc-notched-outline__idle"
  />
</React.Fragment>
`;

exports[`component::NotchedOutline Should match snapshoot cssOnly 1`] = `
<React.Fragment>
  <div
    className="mdc-notched-outline"
  >
    <svg>
      <path
        className="mdc-notched-outline__path"
      />
    </svg>
  </div>
  <div
    className="mdc-notched-outline__idle"
  />
</React.Fragment>
`;
